excel-importer/src/com/axonivy/util/excel/importer/ExcelReader.java

```diff
@@ -30,36 +38,72 @@ private static List<String> getHeaderCellNames(Iterator<Row> rowIterator) {
   }
 
   private static List<Column> createEntityFields(List<String> names, Iterator<Row> rowIterator) {
-    List<Column> columns = new ArrayList<>();
+    Map<String, Column> columnMap = new LinkedHashMap<>();
     if (!rowIterator.hasNext()) {
       return List.of();
     }
-    Row row = rowIterator.next();
-    for(int c = 0; c<row.getLastCellNum(); c++) {
-      var name = names.get(c);
-      var column = toColumn(name, row.getCell(c));
-      columns.add(column);
-    }
-    return columns;
+    rowIterator.forEachRemaining(row -> {
+      for (int cellIndex = 0; cellIndex < row.getLastCellNum(); cellIndex++) {
+        var name = names.get(cellIndex);
+        var cell = row.getCell(cellIndex);
+        if (columnMap.containsKey(name)) {
+          updateColumn(columnMap.get(name), cell);
+        } else {
+          columnMap.put(name, toColumn(name, row.getCell(cellIndex)));
+        }
+      }
+    });
+    return new ArrayList<>(columnMap.values());
   }
 
   private static Column toColumn(String name, Cell cell) {
+    var fieldName = fieldName(name);
```

Comment:
before we analyzed we made the 'name' compatible to dataclass-entity fields, just where we had to create it. Here we were just analyzing the excel structure 
... I think this was more natural to delay that name handling: it tries to handle a problem with entities-classes so it should live where we handle entity-field names.